### ISSUE TITLE
Change the src of the player

### ISSUE THREAD

**Issue by hartman** [brion/ogv.js#334]
While investigating hartman/videojs-ogvjs#7 I discovered that it does not seem to be possible to change the source of the OGVPlayer element. It does not notice it changed, and stopping and starting in order to pick up on this change, seems to result in undefined behavior.

**Comment by brion** [comment]
Hmm, it should work to set src, then call load(). Per spec it's not _supposed_ to pick up the change until you call load(), iirc.

Will test later... probably worth adding this to the test suite too.

**Comment by brion** [comment]
Ok, looks like this works:

```
player.stop();
player.src = 'https://upload.wikimedia.org/wikipedia/commons/transcoded/1/1a/%C2%BFQu%C3%A9_es_Wikipedia%3F.ogv/%C2%BFQu%C3%A9_es_Wikipedia%3F.ogv.160p.ogv';
player.load();
player.play();
```

The explicit stop() call is non-standard and should NOT be necessary. buggggggg

**Comment by hartman** [comment]
> The explicit stop() call is non-standard and should NOT be necessary. buggggggg

I tried it out like that with videojs as well, and it totally confuses the player, since stop will also trigger other async events, that the player will respond to, so you get 2 things that end up being a race condition for what the player will do next.

**Comment by brion** [comment]
Made a stab at this in setsrc branch, might need a few more tweaks. On my iPad on a plane so my testing and compilation options are limited. ;) will test on the ground later.

**Comment by brion** [comment]
Ok, I got that wrong because I misremembered the spec.

> If a src attribute of a media element is set or changed, the user agent must invoke the media element's media element load algorithm. (Removing the src attribute does not do this, even if there are source elements present.)

So basically, this should trigger an implicit .load() on set if I understand.

**Comment by paladox** [comment]
It got fixed in videojs 5.10.1.

See https://github.com/videojs/video.js/pull/3285 please.

Testing with videojs 5.10.1 fixes changing the resolution.
